### DISCUSSION TITLE
Where can I see gravtino’s roadmap?

### DISCUSSION THREAD

**Discussion by zhengsg** [apache/gravitino#5764] [General]
Where can I see gravtino’s roadmap?

**Comment by justinmclean** [comment]
We have a roadmap here and will shortly update it for next year:
https://github.com/apache/gravitino/blob/main/ROADMAP.md

**Comment by akshat0395** [comment]
Hey @justinmclean any updates for the 2025 roadmap? Looking forward to see what in store this year

**Reply by justinmclean** [reply]
We've been a bit slow in updating it sorry, it should get done in the next few weeks.
